ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -435,6 +442,50 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{nam
+                                                            <div className='columns small-3'>SHA:</div>
+                                                            <div className='columns small-9'>
+                                                                <Revision repoUrl={application.spec.source.repoURL} revision={this.state.metadata.revision} />
+                                                            </div>
+                                                        </div>
+                                                    </div>
+                                                    <div className='white-box__details'>
+                                                        <div className='row white-box__details-row'>
+                                                            <div className='columns small-3'>Date:</div>
+                                                            <div className='columns small-9'>
+                                                                <Timestamp date={this.state.metadata.date} />
+                                                            </div>
+                                                        </div>
+                                                    </div>
+                                                    <div className='white-box__details'>
+                                                        <div className='row white-box__details-row'>
+                                                            <div className='columns small-3'>Tags:</div>
+                                                            <div className='columns small-9'>
+                                                                {((this.state.metadata.tags || []).length > 0 && this.state.metadata.tags.join(', ')) || 'No tags'}
+                                                            </div>
+                                                        </div>
+                                                    </div>
+                                                    <div className='white-box__details'>
+                                                        <div className='row white-box__details-row'>
+                                                            <div className='columns small-3'>Author:</div>
+                                                            <div className='columns small-9'>{this.state.metadata.author}</div>
+                                                        </div>
+                                                    </div>
+                                                    <div className='white-box__details'>
+                                                        <div className='row white-box__details-row'>
+                                                            <div className='columns small-3'>Message:</div>
+                                                            <div className='columns small-9'>
```

Comment:
Could we vertically center this? 

<img width="513" alt="Screen Shot 2021-01-26 at 9 06 25 AM" src="https://user-images.githubusercontent.com/6250584/105878339-c8f2b280-5fb5-11eb-87af-9d44a04e0ed5.png">

`style={{display: 'flex', align-items: 'center'}}` on the `columns small-9` div would do it

Reply:
great suggestion! Added